config/cable.yml

```
@@ -2,9 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

prefer non-block version unless you need a block

```suggestion
  url: <%= ENV.fetch("REDIS_URL", "redis://localhost:6379/1") %>
```